docs/pages/_app.js

```diff
@@ -45,7 +45,7 @@ function useFirstRender() {
   return firstRenderRef.current;
 }
 
-acceptLanguage.languages(['en', 'zh', 'pt', 'ru']);
+acceptLanguage.languages(['en', 'zh', 'pt']);
```

Comment:
Does this list all lanugages we support for SSR? If so then we should just re-use that hardcoded list (I think located in `constants.js`) if this is indeed the same concern. 

Otherwise it would be nice to understand how we choose the languages here.

Reply:
Good point, I will update it. After https://github.com/mui-org/material-ui/pull/28663 looks like the `LANGUAGES` and `LANGUAGES_SSR` are identical, but I will change this with `LANGUAGES_SSR` to guard against breaking in the future.

Reply:
It was intentional, I have detailed it here https://github.com/mui-org/material-ui/pull/28869#pullrequestreview-772855646. We should have added a new variable and comment on it.